ketcher-autotests/tests/User-Interface/Floating-windows/floating-windows.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { test } from '@playwright/test';
+import {
+  selectTopPanelButton,
+  TopPanelButton,
+  takeEditorScreenshot,
+  pasteFromClipboardAndAddToCanvas,
+  openFileAndAddToCanvas,
+  openFile,
+  FILE_TEST_DATA,
+  pasteFromClipboard,
+  waitForLoad,
+  pressButton,
+  waitForPageInit,
+  clickInTheMiddleOfTheScreen,
+} from '@utils';
+
+test.describe('Click and drag FG on canvas', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  test('Calculate values (data on canvas)', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-3998
+      Description: verify the floating window with calculated values 
+    */
+    await openFileAndAddToCanvas('Molfiles-V2000/bicycle.mol', page);
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+  });
+
+  test('Calculated values: check accuracy', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-3999(1)
+      Description: verify 0 decimal places after the dot for calculated values 
+    */
+    await openFileAndAddToCanvas('Molfiles-V2000/bicycle.mol', page);
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+    await page
+      .getByRole('listitem')
+      .filter({ hasText: 'Molecular Weight:Decimal places3' })
+      .getByRole('button', { name: '3' })
+      .click();
+    await page.getByRole('option', { name: '0' }).click();
+    await page.getByRole('button', { name: '3', exact: true }).click();
+    await page.getByRole('option', { name: '0' }).click();
+  });
+
+  test('Calculated values: check accuracy 2', async ({ page }) => {
+    /*
+      Test case: EPMLSOPKET-3999(2)
+      Description: verify 7 decimal places after the dot for calculated values 
+    */
+    await openFileAndAddToCanvas('Molfiles-V2000/bicycle.mol', page);
+    await selectTopPanelButton(TopPanelButton.Calculated, page);
+    await page
```

Comment:
I see, there is a small code duplication in this test and above. That's not critical, if it is only a matter of two tests. But if more test will be written, it would be good to have a function.
Anyway, I would propose to create a function like:
```
// it is required to find a better name, b/c I don't know which parameters you set
async function setSomeParameters(page: Page, param1, param2, ...) {
await page
      .getByRole('listitem')
      .filter({ hasText: 'Molecular Weight:Decimal places3' })
      .getByRole('button', { name: '3' })
      .click();
    await page.getByRole('option', { name: '0' }).click();
    await page.getByRole('button', { name: '3', exact: true }).click();
    await page.getByRole('option', { name: '0' }).click();
}
```